content/en/docs/ops/best-practices/image-signing-validation/index.md

```diff
@@ -0,0 +1,67 @@
+---
+title: Image Signing And Validation
+description: Describes how to use image signatures to verify the provenance of Istio images.
+weight: 35
+aliases:
+    - /docs/ops/best-practices/image-signing-validation/
+keywords: [install]
+owner: istio/wg-environments-maintainers
+test: n/a
+---
+
+This page describes how to use [Cosign](https://github.com/sigstore/cosign) to
+validate the provenance of Istio image artifacts.
+
+Cosign is a tool developed as part of the
+[sigstore](https://www.sigstore.dev/https://www.sigstore.dev/) project which
+simplifies signing and validation of signed Open Container Initiative artifacts,
+such as docker images.
+
+In Istio, we sign all officially published docker images as part of our release
+process using our own private key. End users can then verify these images using
```

Comment:
```suggestion
process using our own private key starting with Istio 1.12. End users can then verify these images using
```

Reply:
We may actually be able to say 1.11.x (not sure what x is).